packages/neo-one-smart-contract-lib/src/__tests__/ownership/TestOwnableContract.test.ts

```diff
@@ -0,0 +1,80 @@
+import { common, crypto } from '@neo-one/client-common';
+import { SmartContractAny } from '@neo-one/client-core';
+import { withContracts } from '@neo-one/smart-contract-test';
+import * as path from 'path';
+
+const RECIPIENT = {
+  PRIVATE_KEY: '7d128a6d096f0c14c3a25a2b0c41cf79661bfcb4a8cc95aaaea28bde4d732344',
+  PUBLIC_KEY: '02028a99826edc0c97d18e22b6932373d908d323aa7f92656a77ec26e8861699ef',
+};
+
+describe('Ownable', () => {
+  test('deploy + transfer', async () => {
+    await withContracts<{ testOwnable: SmartContractAny }>(
+      [
+        {
+          filePath: path.resolve(__dirname, '..', '..', '__data__', 'contracts', 'ownership', 'TestOwnable.ts'),
+          name: 'TestOwnable',
+        },
+      ],
+      async ({ client, networkName, testOwnable: smartContract, masterAccountID }) => {
+        crypto.addPublicKey(
+          common.stringToPrivateKey(RECIPIENT.PRIVATE_KEY),
+          common.stringToECPoint(RECIPIENT.PUBLIC_KEY),
+        );
+
+        const deployResult = await smartContract.deploy(masterAccountID.address, { from: masterAccountID });
+
```

Comment:
nit: remove newline - the line above is organizationally/logically related to the following lines.